docs/content/tools/script-overviews/_index.md.old

```diff
@@ -0,0 +1,426 @@
+---
+title: Overview and Usage of APRL Scripts
```

Comment:
```suggestion
---
title: Overview and Usage of APRL Scripts
```
Can we just get rid of this file?

Reply:
I think so but I wasn't entirely sure so I wanted to keep it around in case there was something in it that was needed. Can we remove it next week if we don't find we're missing anything?